src/main/java/racingcar/service/RoundService.java

```diff
@@ -0,0 +1,34 @@
+package racingcar.service;
+
+import racingcar.domain.Car;
+import racingcar.domain.Cars;
+import racingcar.domain.RandomNumber;
+
+public class RoundService {
+    private static final int MOVING_FORWARD = 4;
+
+    public void playRound(Cars cars) {
+        for (Car car : cars.getCars()) {
+            int randomValue = generateRandomValue();
+            moveCarIfNeeded(car, randomValue);
+
+        }
+    }
+
+    private int generateRandomValue() {
+        return RandomNumber.make();
+    }
+
+    private void moveCarIfNeeded(Car car, int randomValue) {
+        if (shouldMove(randomValue)) {
+            car.forward();
+        }
+    }
+
+    private boolean shouldMove(int randomValue) {
+        if (randomValue >= MOVING_FORWARD) {
+            return true;
+        }
+        return false;
```

Comment:
조건문을 직접 반환하는 것은 어떨까요?
```suggestion
        return randomValue >= MINIMUM_MOVING_FORWARD_VALUE;
```